src/ExpressionEvaluator/Core/Source/Concord/Concord.csproj

```diff
@@ -31,8 +31,12 @@
     <None Include="packages.config" />
```

Comment:
Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declation.